README.md

```diff
@@ -45,10 +45,11 @@ This charm offers integration with [kratos-operator](https://github.com/canonica
 
 The following instructions assume that you have deployed `traefik-admin` and `traefik-public` charms and related them to hydra.
 
-If you have deployed [Login UI charm](https://github.com/canonical/identity-platform-login-ui), you can configure it with hydra by providing its URL.
+If you have deployed [Login UI charm](https://github.com/canonical/identity-platform-login-ui), you can configure it with hydra by using eachother's integrations.
```

Comment:
Could you split and modify the `Integration with Kratos and UI` paragraph? Also add the relation names (to avoid ambiguity) and replace `relate` with `integrate`. So sth like:

### Kratos

This charm offers integration with [kratos-operator](https://github.com/canonical/kratos-operator). In order to integrate hydra with kratos, it needs to be able to access hydra's admin API endpoint.
To enable that, integrate the two charms:
```console
juju integrate kratos hydra
```

### Identity Platform Login UI

This charm offers integration with [identity-platform-login-ui-operator](https://github.com/canonical/identity-platform-login-ui-operator).
In order to integrate them, run:
```console
juju integrate hydra:ui-endpoint-info identity-platform-login-ui-operator:ui-endpoint-info
juju integrate identity-platform-login-ui-operator:hydra-endpoint-info hydra:hydra-endpoint-info
```